src/components/Patient/SampleViewAdmin.tsx

```diff
@@ -117,22 +117,31 @@ export default function SampleViewAdmin() {
     });
   };
 
-  const parseExportData = (data: string) =>
-    data
-      .trim()
-      .split("\n")
-      .map((row: string) =>
-        row
-          .trim()
-          .split(",")
-          .map((field: string) =>
-            new Date(field).toString() === "Invalid Date"
-              ? field
-              : formatDateTime(field),
-          )
-          .join(","),
-      )
-      .join("\n");
+  const parseExportData = (data: string) => {
+    const rows = data.trim().split("\n");
```

Comment:
btw cleaner approach:

```suggestion
    const [header, ...rows] = data.trim().split("\n");
```

Reply:
This requested change is done.